src/EA.Weee.Web/Areas/AatfReturn/Views/App_LocalResources/ReturnsResources.resx

```diff
@@ -198,6 +213,9 @@
   <data name="ReusedAsWholeAppliance" xml:space="preserve">
     <value>Reused as a whole appliance</value>
   </data>
+  <data name="Saveandcontinue" xml:space="preserve">
```

Comment:
@sreedhar-civica I added a SaveAndContinue resource could you update and remove this one?